docs/installation.md

```diff
@@ -63,15 +63,16 @@ in your [`PATH`](https://en.wikipedia.org/wiki/PATH%5F%28variable%29) or add the
 | Linux     | [32 bit][linux32]    | [64 bit][linux64]    |
 | Linux ARM | [32 bit][linuxarm32] | [64 bit][linuxarm64] |
 | Windows   | [32 bit][windows32]  | [64 bit][windows64]  |
-| macOS     |                      | [64 bit][macos]      |
+| macOS     | [ARM][macosarm64]    | [64 bit][macos64]    |
```

Comment:
```suggestion
| macOS     |                      | [64 bit][macos64]    |
| macOS ARM |                      | [64 bit][macosarm64] |
```
I think the table is easier to understand if it follows a consistent format (as previously established by the Linux rows.